test/Makefile

```diff
@@ -25,13 +25,13 @@ build:
 test: run k8s
 
 run:
-	ginkgo --focus "Runtime" -v -- --cilium.provision=$(PROVISION) --cilium.registryCredentials=$(REGISTRY_CREDENTIALS)
+	ginkgo --focus "Runtime" --tags integration_tests -v -- --cilium.provision=$(PROVISION) --cilium.registryCredentials=$(REGISTRY_CREDENTIALS)
```

Comment:
Do we need to do this for the `build` target as well?

Upstream docs say:

```
Build flags:
    -tags
        Set -tags to pass in build tags down to the compilation step.
```

Reply:
No, we don't need to add `-tags=integration_tests` to the build target as the tag is only used in tests.

Reply:
> One minor thing to check on the `make -C test build` target.

Ah, I see what you mean now. AFAICT the resulting `test.test` binary is not used anywhere, but I've updated the target to add the `integration_tests` tag.